### PR TITLE
Refactored recipe so that only one method needs be called to add a mission json

### PR DESCRIPTION
**Pull request by kaitlyndlee**
Added  addMissionJson that combines the two method calls that each script was doing. Removed getRecipeJSON since no files are using it now. Left in AddJsonFile since ServiceJobber is using it.

closes https://github.com/USGS-Astrogeology/PDS-Pipelines/issues/313

### CODE REVIEW

File: pds_pipelines/Recipe.py [hunk 1]
```diff
@@ -47,30 +47,25 @@ def AddJsonFile(self, file, proc):
...
-        """
-        return self.recipe
-
-    def getRecipeJSON(self, mission):
+    def addMissionJson(self, mission, proc):
         """
         Parameters
         ----------
         mission : str
+        proc : str
```
Comment (acpaquette) [inline pds_pipelines/Recipe.py]: Could the doc string be updated for addMission and AddJsonFile?

File: pds_pipelines/Recipe.py [hunk 2]
```diff
@@ -47,30 +47,25 @@ def AddJsonFile(self, file, proc):
...
-        """
-        Returns
-        -------
-        list
-            self.recipe
-        """
-        return self.recipe
-
-    def getRecipeJSON(self, mission):
+    def addMissionJson(self, mission, proc):
```
Comment (acpaquette) [inline pds_pipelines/Recipe.py]: Not to be a nudge but it almost seems like this function doesn't need to exist if all it's doing is handling paths. Are we using the AddJsonFile function anywhere else?

Comment (kaitlyndlee) [reply pds_pipelines/Recipe.py]: Yes, we are using it in ServiceJobber.py.

**Comment by acpaquette** [comment]
Could we update the doc strings to be more like the doc string in getPDSid in UPCprocess? Short description of the function and a short description of each argument

**Pull request merged**